docs/guide/deployment.md

```diff
@@ -198,21 +200,21 @@ See [Layer0 Documentation > Framework Guides > VuePress](https://docs.layer0.co/
 
 1. On [Netlify](https://netlify.com), set up a new project from GitHub with the following settings:
 
-    - **Build Command:** `yarn docs:build`
-    - **Publish directory:** `docs/.vuepress/dist`
+   - **Build Command:** `npm i -g pnpm@7 && pnpm docs:build`
```

Comment:
Is it necessary to add `@7` for pnpm? If not, we'd better remove it so we don't need to update it in the future